nemo_skills/pipeline/utils/cluster.py

```diff
@@ -197,6 +197,18 @@ def get_env_variables(cluster_config):
             if env_var_name not in _logged_optional_env_vars:
                 LOG.info(f"Optional environment variable {env_var_name} not found in user environment; skipping.")
                 _logged_optional_env_vars.add(env_var_name)
+
+    # replace placeholders with actual env var values
+    for key, value in env_vars.items():
```

Comment:
Perhaps, we should have a clarifying comment that these env vars are only resolved on the host submitting the job, not the host where the jobs are run?

Reply:
Done